frontend/BoardGUI.py

```diff
@@ -1,20 +1,29 @@
 import pygame as pg
-# from backend.Board import Board
+
+from backend.Board import Board
 from backend.GameManager import GameManager
-# import backend.Cell as CellType
+from backend.Piece import Turn
```

Comment:
Hmm, maybe Turn should have its own file then, just to better separate everything...

Reply:
Agreed

Reply:
https://github.com/Advanced-Software-Engineering-UMKC/Nine-Mens-Morris/issues/32